examples/browser-verify/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "rust-webpack-template"
+description = "My super awesome Rust, WebAssembly, and Webpack project!"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
+
+[features]
+# If you uncomment this line, it will enable `wee_alloc`:
+#default = ["wee_alloc"]
+
+[dependencies]
+# The `console_error_panic_hook` crate provides better debugging of panics by
```

Comment:
Are most of these comments from the webpack template? Should we include them in our example as well?